### ISSUE TITLE
Wrong Line breaking

### ISSUE THREAD

**Issue by kmerlotti** [skia4delphi/skia4delphi#112]
Line breaking isn't working as expected:

See the listview items with Skia **Disabled**:
![LineBreakingSkiaDisabled](https://user-images.githubusercontent.com/2359006/170339419-df96b8e0-1a22-4692-88a0-9bc65ffdcb1e.jpg)

And now with Skia **Enabled** (where is the third line?):
![LineBreakingSkiaEnabled](https://user-images.githubusercontent.com/2359006/170339427-603c5bc0-0bd8-45b9-84c8-db258b3293c1.jpg)

The linebreaking is done with TField.onGetText event, or adding listview items manually. The code is something like this:
`finalString := string1 + sLineBreak + string2;`

Here is the full sample that generated the images above: [SkiaLineBreakingD11.0Updt1.zip](https://github.com/skia4delphi/skia4delphi/files/8773652/SkiaLineBreakingD11.0Updt1.zip). It was tested on Delphi 10.4.2 and D11.1, on Windows and on Android.

Let me know if I can help with something else and, by the way, my congratulations to the project members for the excellent work!

Best regards,
KM

**Comment by viniciusfbb** [comment]
Hi @kmerlotti, thank you so much for reporting the issue!

I confirmed it here too and in fact Skia4Delphi is showing a different result from the standard FMX.

This is happening because the ListView.ItemAppearence.Item.Detail has WordWrap False. When selecting WordWrap True, both will have the same result.
![image](https://user-images.githubusercontent.com/11139086/170500226-e640a1be-7c2b-4b89-96cd-02c7e05991ff.png)

Basically this happens because when informing WordWrap False, Skia4Delphi will limit the text to only 1 line, and the livebindings and the manual code, are inserting the line break in the detail text.

I'm wondering now if WordWrap would always limit it to 1 line (as Skia4Delphi is doing) or limit it to 1 line except in explicit line breaks (like FMX does).

**Comment by kmerlotti** [comment]
Thank you very much for your fast support, @viniciusfbb!

I can confirm that enabling the WordWrap propery makes line breaking work as expected. 

But, I'd suggest you (the team) to review the Skia4Delphi behavior, as you mentioned, in order to make line breaking by code to work regardless of the WordWrap property. I think the property is more tied to automaticaly line breaking, and the code must override any automatically behavior.

By the way, in my case (the reall app, not the sample attached to this issue), enable the WordWrap property can make last lines invisible inside a single item, because the listview items has fixed size, so, if the first detail line breaks by the wordwrap, the last line won't be shown. Using WordWrap disabled, I can guarantee all the lines will be visible (inside the item bounds). I know this can be only my case, but I'm just sharing to let you know a real case that might empower my suggestion above.

Thanks again!

**Comment by PonyPC** [comment]
I have same problem, at first, I thought it might be caused by my third-party component. Multiple lines only show the first line.

**Comment by viniciusfbb** [comment]
Right. We will implement the behavior identical to FMX, already in the next release.

**Comment by viniciusfbb** [comment]
Fixed in our new release [3.4.1](https://github.com/skia4delphi/skia4delphi/releases)
